README.md

```diff
@@ -170,6 +170,34 @@ jobs:
       - run: $GOPATH/bin/buildevents cmd $TRAVIS_BUILD_ID $STEP_SPAN_ID go-test -- go test -timeout 2m -mod vendor ./...
 ```
 
+## Attaching more traces from your build and test process
+
+Every command running through `buildevents cmd` will receive a `HONEYCOMB_TRACE` environment variable that contains a marshalled trace propagation context. This can be used to connect more spans to this trace.
+
+Ruby Beeline example:
+```ruby
+# at the very start of the command
+# establish a command-level span, linking to the buildevent
+process_span = Honeycomb.start_span(name: File.basename($PROGRAM_NAME), serialized_trace: ENV['HONEYCOMB_TRACE'])
+Honeycomb.add_field_to_trace('process.full_name', $PROGRAM_NAME)
+Honeycomb.add_field_to_trace('process.args', ARGV)
+
+# override the HONEYCOMB_TRACE for sub-processes
+ENV['HONEYCOMB_TRACE'] = process_span.to_trace_header unless ENV['HONEYCOMB_TRACE']
```

Comment:
My understanding of this line is that `ENV['HONEYCOMB_TRACE']` will only be set to the new trace header if the environment variable is not already set, the equivalent of ...

```ruby
ENV['HONEYCOMB_TRACE'] ||= process_span.to_trace_header
```

The mention of "override" in the comment has me thinking the expected behavior will be to set the starting trace context for child processes to the trace information modified above in the parent process. But I don't think that is what is happening here.

Reply:
oof, you're right! And I was worried about my golang skills :sweat_smile: